### ISSUE TITLE
Remove deprecated t.words.protocols.irc.IRCClient.me

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5059]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5059|
|Type|task|
|Created|2011-04-20 21:58:15Z|
|Branch|https://github.com/twisted/twisted/tree/ircclient-me-5059|

`IRCClient.me` in [twisted/words/protocols/irc.py](twisted/words/protocols/irc.py) was deprecated in Twisted 9.0 ([#3910](https://github.com/twisted/twisted/issues/3910)), and can be removed.

<details><summary>Searchable metadata</summary>

```
trac-id__5059 5059
type__task task
reporter__thijs thijs
priority__low low
milestone__ 
branch__branches_ircclient_me_5059 branches/ircclient-me-5059
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__words words
keywords__ 
time__1303336695000000 1303336695000000
changetime__1318865868000000 1318865868000000
version__None None
owner__thijs thijs
cc__thijs cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [32826]) Branching to 'ircclient-me-5059'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [32827]) Remove deprecated IRCClient.me, refs [#5059](https://github.com/twisted/twisted/issues/5059)

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_5"></a>|@jonathanj set owner to @thijstriemstra|
|-|-|

Thanks for cleaning up a bunch of those ugly docstrings, I think you can go ahead and merge this.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [32899]) Merge ircclient-me-5059: Remove t.w.protocols.irc.IRCClient.me, deprecated in v9.0. Use IRCClient.describe instead.

Author: thijs
Reviewer: jonathanj
Fixes: [#5059](https://github.com/twisted/twisted/issues/5059)
